consensus/block.go

```diff
@@ -41,7 +40,7 @@ func (m *consensusModule) prepareBlock() (*types.Block, error) {
 		Height:            int64(m.Height),
 		Hash:              hex.EncodeToString(appHash),
 		NumTxs:            uint32(len(txs)),
-		LastBlockHash:     typesGenesis.GetNodeState(nil).AppHash, // testing temporary
+		LastBlockHash:     m.appHash,
 		ProposerAddress:   m.privateKey.Address(),
 		QuorumCertificate: nil,
 	}
```

Comment:
On line 63 here, do we have a high degree of confidence this call is ok through testing?

Reply:
**_Temporarily accepted._**

Reply:
TODO(phthan0): Fix it.